src/promptflow/promptflow/_sdk/_pf_client.py

```diff
@@ -101,6 +101,7 @@ def stream(self, run: Union[str, Run]) -> Run:
         :param run: Run object or name of the run.
         :type run: Union[str, ~promptflow.sdk.entities.Run]
         :return: flow run info.
+        :rtype: ~promptflow.sdk.entities.Run
```

Comment:
Will this function return None?

Reply:
For currently implementation, it will not. If no run is found, it will raise error in get function

Reply:
"Note that you can combine the ~ and . prefixes: :py:meth:`~.TarFile.close` will reference the tarfile.TarFile.close() method, but the visible link caption will only be close()."